compiler/compile/OMRCompilation.hpp

```diff
@@ -328,6 +329,8 @@ class OMR_EXTENSIBLE Compilation
 
    bool trace(OMR::Optimizations o)             { return _options->trace(o); }
    bool isDisabled(OMR::Optimizations o)        { return _options->isDisabled(o); }
+   TR::RelocationRuntime* reloRuntime() { return _reloRuntime;}
```

Comment:
This change may work, but it will be pretty confusing in OpenJ9, where J9::Compilation *also* defines an instance function called `reloRuntime()` but that one returns `TR_RelocationRuntime *`...

Reply:
Thank you, I have renamed the changes.